Libraries/pbxbuild/Sources/Target/Environment.cpp

```diff
@@ -269,8 +304,8 @@ Create(Build::Environment const &buildEnvironment, Build::Context const &buildCo
             projectActionEnvironment.insertFront(level, false);
         }
 
-        projectConfigurationFile = LoadConfigurationFile(projectConfiguration, projectActionEnvironment);
-        if (projectConfigurationFile != nullptr) {
+        projectConfigurationFile = LoadConfigurationFile(Filesystem::GetDefaultUNSAFE(), projectActionEnvironment, workingDirectory, projectConfiguration);
```

Comment:
Since you're using this further down, too, maybe cache the Unsafe filesystem in a local variable?